src/elements/types/ExportAllDeclaration.js

```diff
@@ -0,0 +1,26 @@
+import ModuleDeclaration from '../ModuleDeclaration';
+
+export default class ExportAllDeclaration extends ModuleDeclaration {
+    constructor(childNodes) {
+        super('ExportAllDeclaration', childNodes);
+    }
+
+    _acceptChildren(children) {
+        children.passToken('Keyword', 'export');
+        children.skipNonCode();
+        children.passToken('Punctuator', '*');
+        children.skipNonCode();
+        children.passToken('Identifier', 'from');
+        children.skipNonCode();
+        let source = children.passNode('Literal');
+        children.skipNonCode();
+        children.skipSemicolon();
+        children.assertEnd();
```

Comment:
Calling `skipNonCode()` and `skipSemicolon()` in a row means we can have trailing spaces.

I would suggest something like this:

```
let source = children.passNode('Literal');
if (!children.isEnd) {
    children.skipNonCode();
    children.passToken('Punctuator', ';');
}
children.assertEnd();
```